app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/check_your_details.scala.html

```diff
@@ -261,7 +263,8 @@ <h1 class="heading-large">@messages("cds.form.check-answers")</h1>
                         "contact-details",
                         messages("cds.form.customs-contact"),
                         contactDetailsHtml(cd),
-                        Some(uk.gov.hmrc.eoricommoncomponent.frontend.controllers.subscription.routes.ContactDetailsController.reviewForm(service, journey)),
+                        if (journey == Journey.Register) Some(uk.gov.hmrc.eoricommoncomponent.frontend.controllers.registration.routes.ContactDetailsController.reviewForm(service))
```

Comment:
please see if this is accessible via migration

Reply:
It looks like this check your answers can be accessed only in subscription journey, so removing if to check register